minecraft-quilt/minecraft-quilt.json

```diff
@@ -0,0 +1,115 @@
+{
+  "type": "minecraft-java",
+  "display": "Quilt - Minecraft",
+  "data": {
+    "eula": {
+      "type": "boolean",
+      "desc": "Do you (or the server owner) agree to the <a href='https://account.mojang.com/documents/minecraft_eula'>Minecraft EULA?</a>",
+      "display": "EULA Agreement (true/false)",
+      "required": true,
+      "value": "false"
+    },
+    "game-version": {
+      "type": "string",
+      "desc": "Version of Minecraft to install (Oldest quilt version is 1.14)",
+      "display": "GAME VERSION",
+      "required": true,
+      "value": "1.19.3"
+    },
+    "ip": {
+      "type": "string",
+      "desc": "What IP to bind server to",
+      "display": "IP",
+      "required": true,
+      "value": "0.0.0.0"
+    },
+    "java": {
+      "type": "string",
+      "desc": "\"java\" for system standard, else direct path to java",
+      "display": "path to java executable",
+      "required": true,
+      "value": "java",
+      "userEdit": true
+    },
+    "memory": {
+      "type": "integer",
+      "desc": "How much memory in MB to allocate to the Java Heap",
+      "display": "MEMORY (MB)",
+      "required": true,
+      "value": "1024"
+    },
+    "motd": {
+      "type": "string",
+      "desc": "This is the message that is displayed in the server list of the client, below the name. The MOTD does support <a href='https://minecraft.gamepedia.com/Formatting_codes' target='_blank'>color and formatting codes</a>.",
+      "display": "MOTD message of the day",
+      "required": true,
+      "value": "A Minecraft Server\\n\\u00A79 hosted on PufferPanel"
+    },
+    "port": {
+      "type": "string",
+      "desc": "What port to bind server to",
+      "display": "PORT",
+      "required": true,
+      "value": "25565"
+    }
+  },
+  "install": [
+    {
+      "files": [
+        "https://maven.quiltmc.org/repository/release/org/quiltmc/quilt-installer/latest/quilt-installer-latest.jar"
+      ],
+      "type": "download"
+    },
+    {
+      "source": "quilt-installer-*.jar",
+      "target": "quilt-installer.jar",
+      "type": "move"
+    },
+    {
+      "commands": [
+        "${java} -jar quilt-installer.jar install server ${game-version} --download-server --install-dir=."
+      ],
+      "type": "command"
+    },
+    {
+      "target": "server.properties",
+      "text": "server-ip=${ip}\nserver-port=${port}\nmotd=${motd}",
+      "type": "writefile"
+    },
+    {
+      "target": "eula.txt",
+      "text": "eula=${eula}",
+      "type": "writefile"
+    },
+    {
+      "source": "quilt-server-launch.jar",
+      "target": "quilt-server.jar",
+      "type": "move"
+    },
+    {
+      "target": "mods",
+      "type": "mkdir"
+    },
+    {
+      "target": "installer",
+      "type": "mkdir"
+    },
+    {
+      "source": "quilt-installer.jar",
+      "target": "installer/quilt-installer.jar",

```

Comment:
Any reason to keep the installer instead of deleting it?

Reply:
No can be deleted indeed. This file was based on the Fabric config and it was done over there previously. That's why I kept it in